modules/apps/frontend-data-set/frontend-data-set-api/src/main/java/com/liferay/frontend/data/set/view/table/BaseDateFDSTableSchemaField.java

```diff
@@ -16,17 +17,31 @@ public JSONObject getFormat() {
 		return _formatJSONObject;
 	}
 
+	public String getTimeZoneBehavior() {
+		return _timeZoneBehavior;
+	}
+
 	public void setFormat(JSONObject formatJSONObject) {
 		_formatJSONObject = formatJSONObject;
 	}
 
+	public void setTimeZoneBehavior(String timeZoneBehavior) {
+		_timeZoneBehavior = timeZoneBehavior;
+	}
+
 	@Override
 	public JSONObject toJSONObject() {
 		JSONObject jsonObject = super.toJSONObject();
 
-		return jsonObject.put("format", getFormat());
+		return jsonObject.put(
+			"format", getFormat()
+		).put(
+			"timeZoneBehavior", getTimeZoneBehavior()
+		);
```

Comment:
In difference to previous version, I set time zone behavior outside of `format`. The idea is that `format` structure matches second argument of `Intl.DateTimeFormat`.